seed/php-sdk/imdb/tests/Seed/Types/ExampleTypeTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Seed\Types;
```

Comment:
Heads up, we'll want to use `Seed\Tests\...` in the `tests/` directory here (e.g. `Seed\Tests\Types`). Nothing actionable here, just FYI once you revisit this in the generator. 

This will naturally be fixed for you once you use the [getTestsNamespace](https://github.com/fern-api/fern/blob/7f623dd4d5699493e0171c0c019d970eec36e93b/generators/php/codegen/src/context/AbstractPhpGeneratorContext.ts#L31) function.